testdata/p4_16_errors/key-name.p4

```diff
@@ -34,4 +34,5 @@ control compute(inout hdr h) {
     }
 }
 
+/* P4TEST_IGNORE_STDERR */
```

Comment:
If you merge my change into your branch, I'll then squash&merge pull the changes into master.